pkg/vulnsrc/python/python.go

```diff
@@ -136,3 +137,24 @@ func (vs VulnSrc) Get(pkgName string) ([]Advisory, error) {
 	}
 	return results, nil
 }
+
+func decodeAdvisoryDB(r io.Reader) (AdvisoryDB, error) {
+	var obj map[string]interface{}
+	advisoryDB := AdvisoryDB{}
+	if err := json.NewDecoder(r).Decode(&obj); err != nil {
+		return nil, xerrors.Errorf("failed to decode JSON: %w", err)
+	}
+	for k, v := range obj {
+		slice, ok := v.([]interface{})
+		if !ok {
+			continue
+		}
```

Comment:
```suggestion
		if k == "$meta" {
			continue
		}
```

This is more readable to me. I know your implementation can handle when other data structures are added in the future. But if `$meta` will be changed to slice, an error occurs. Both implementations have pros and cons. If a prefix `$` always means metadata, we can write it as follows.

```
		if strings.HasPrefix(k) == "$" {
			continue
		}
```

At the moment, we can just skip `$meta`.

Reply:
ok